webapp/package.json

```diff
@@ -17,34 +17,34 @@
     "npm": ">=6.4.1"
   },
   "dependencies": {
-    "@angular/animations": "^12.2.1",
-    "@angular/common": "^12.2.1",
-    "@angular/compiler": "^12.2.1",
-    "@angular/core": "^12.2.1",
-    "@angular/forms": "^12.2.1",
-    "@angular/platform-browser": "^12.2.1",
-    "@angular/platform-browser-dynamic": "^12.2.1",
-    "@angular/router": "^12.2.1",
+    "@angular/animations": "^12.2.16",
```

Comment:
Maybe test that app navigation is okay, QA don't AT upgrade lib tickets. 
I think one ticket for Allies (any can do it actually) would be to upgrade to latest angular (as today v14) not sure if there's breaking changes